sonar-text-plugin/src/main/resources/org/sonar/plugins/secrets/configuration/zuplo.yaml

```diff
@@ -0,0 +1,27 @@
+provider:
+  metadata:
+    name: Zuplo
+    category: Development
+    message: Make sure this Zuplo API key gets revoked, changed, and removed from the code.
+
+  rules:
+    - id: zuplo-api-key
+      rspecKey: S6753
+      metadata:
+        name: Zuplo API keys should not be disclosed
+      detection:
+        pre:
+          include:
+            content:
+              - zpka_
+        matching:
+          pattern: "\\b(zpka_[a-zA-Z\\d]{32}_[a-zA-Z\\d]{8})\\b"
+      examples:
+        - text: |
+            Authorization: Bearer zpka_213d294a9a5a44619cd6a02e55a20417_5f43e4d0
```

Comment:
Beware this is a real world token. I am not too sure what to think about legal matters here, but in doubt, I would generate a fake one.

Reply:
I will modify the token